test/test_manifest.py

```diff
@@ -31,6 +31,8 @@ def find_image_size_from(manifest_str):
 
 @pytest.mark.parametrize("tc", gen_testcases("manifest"))
 def test_manifest_smoke(build_container, tc):
+    testutil.pull_container(tc.container_ref, tc.target_arch)
```

Comment:
Hm, would it make sense to make this part of the build_container fixture somehow? it seems to repeat quite a bit.

Reply:
Yeah good idea, let me try

Reply:
Would it maybe make sense to make a different fixture for some of these tests rather? It doesn't look like the `build_container` fixture should be used here, right?